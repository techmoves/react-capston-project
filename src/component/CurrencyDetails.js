import React, { useEffect } from 'react';
import { useNavigate, useParams } from 'react-router-dom';
import { useDispatch, useSelector } from 'react-redux';
import { getCurrency } from '../redux/Currencies/myStockcurrenciesSlice';
import './style.css';
import back from '../images/arrow-8.png';
import mic from '../images/microphone.png';
import setting from '../images/gear.png';
import right from '../images/arrow-4.png';

function CurrencyDetails() {
  const { symbols } = useParams();
  const dispatch = useDispatch();
  const navigate = useNavigate();
  const { currencies, currency } = useSelector((store) => store.currencies);
  let currencyDetail = '';
  currencies.forEach((curr) => {
    if (curr.symbol === symbols) {
      currencyDetail = curr.name;
    }
  });

  useEffect(() => {
    dispatch(getCurrency(symbols));
  }, [dispatch]);

  return (
    <div className="mydetail">
      <div className="input">
        <button type="button" onClick={() => navigate('/')}>
          <img src={back} alt="back-option-button" />
        </button>
        <input type="text" />
        <button type="button">
          <img src={mic} alt="microphone-button" />
        </button>
        <button type="button">
          <img src={setting} alt="option-button" />
        </button>
      </div>
      <ul className="myList">
        <li>
          {' '}
          One
          {currencyDetail}
          {' '}
          is worth
        </li>
        <li>
          <span>{currency.usd}</span>
          {' '}
          <span className="the-currency">US dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>
            {currency.cad}
            {' '}
          </span>
          {' '}
          <span className="the-currency">Canadian dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>
            {currency.bhd}
            {' '}
          </span>
          {' '}
          <span className="the-currency">Bahraini dinar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.kwd}</span>
          {' '}
          <span className="the-currency">Kuwaiti dinar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.pln}</span>
          {' '}
          <span className="the-currency">Polish z≈Çoty</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.thb}</span>
          {' '}
          <span className="the-currency">Thai baht</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ils}</span>
          {' '}
          <span className="the-currency">Israeli new shekel</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.idr}</span>
          {' '}
          <span className="the-currency">Indonesian rupiah</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.czk}</span>
          {' '}
          <span className="the-currency">Czech koruna</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.eur}</span>
          {' '}
          <span className="the-currency">euros</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.btc}</span>
          {' '}
          <span className="the-currency">bitcoin</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.chf}</span>
          {' '}
          <span className="the-currency">Swiss franc</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.sek}</span>
          {' '}
          <span className="the-currency">Swedish krona</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.nok}</span>
          {' '}
          <span className="the-currency">Norwegian Krone</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.dkk}</span>
          {' '}
          <span className="the-currency">Danish Krone</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.aud}</span>
          {' '}
          <span className="the-currency">Australian dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.nzd}</span>
          {' '}
          <span className="the-currency">New Zealand dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.hkd}</span>
          {' '}
          <span className="the-currency">Hong Kong dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.twd}</span>
          {' '}
          <span className="the-currency">New Taiwan dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.sgd}</span>
          {' '}
          <span className="the-currency">Singapore dollars</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.cny}</span>
          {' '}
          <span className="the-currency">Chinese renminbi</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.jpy}</span>
          {' '}
          <span className="the-currency">Japanese yen</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.krw}</span>
          {' '}
          <span className="the-currency">South Korean won</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.inr}</span>
          {' '}
          <span className="the-currency">Indian rupee</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.mxn}</span>
          {' '}
          <span className="the-currency">Mexican peso</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.brl}</span>
          {' '}
          <span className="the-currency">Brazilian real</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.kyd}</span>
          {' '}
          <span className="the-currency">Cayman Island dollar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.gip}</span>
          {' '}
          <span className="the-currency">Gibraltar pound</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.jod}</span>
          {' '}
          <span className="the-currency">Jordanian dinar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.omr}</span>
          {' '}
          <span className="the-currency">Omani rial</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>

        <li>
          <span>{currency.aed}</span>
          {' '}
          <span className="the-currency">UAE dirham</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.try}</span>
          {' '}
          <span className="the-currency">Turkish lira</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.huf}</span>
          {' '}
          <span className="the-currency">Hungarian forint</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.clp}</span>
          {' '}
          <span className="the-currency">Chilean peso</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.sar}</span>
          {' '}
          <span className="the-currency">Saudi riyal</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.php}</span>
          {' '}
          <span className="the-currency">Philippine peso</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.myr}</span>
          {' '}
          <span className="the-currency">Malaysian ringgit</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.cop}</span>
          {' '}
          <span className="the-currency">Colombian peso</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.rub}</span>
          {' '}
          <span className="the-currency">Russian ruble</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ron}</span>
          {' '}
          <span className="the-currency">Romanian leu</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.pen}</span>
          {' '}
          <span className="the-currency">Peruvian sol</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.bgn}</span>
          {' '}
          <span className="the-currency">Bulgarian lev</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">South African Rand</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">Mozambican Medicash</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">Zim Dollar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">Argentine peso</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">Saudi Arabia Kuwaiti dinar</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">CHH</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">NZD</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
        <li>
          <span>{currency.ars}</span>
          {' '}
          <span className="the-currency">AUD</span>
          <img className="more-details" src={right} alt="right-arrow" />
        </li>
      </ul>
    </div>
  );
}

export default CurrencyDetails;
