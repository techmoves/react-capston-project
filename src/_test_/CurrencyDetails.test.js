import '@testing-library/jest-dom';
import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { configureStore } from '@reduxjs/toolkit';
import myCurrencies from './Currencies';
import CurrencyDetails from '../component/CurrencyDetails';
import myCurrency from './Currency';

const reducer = (
  state = {
    currencies: { currencies: myCurrencies, currency: myCurrency },
  },
) => state;

const store = configureStore({ reducer });
describe('Render Test', () => {
  it('Component renders correctly', () => {
    const mytree = render(
      <MemoryRouter>
        <Provider store={store}>
          <CurrencyDetails />
        </Provider>
      </MemoryRouter>,
    );
    expect(mytree).toMatchSnapshot();
  });
});
