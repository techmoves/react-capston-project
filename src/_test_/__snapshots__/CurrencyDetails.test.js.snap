// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test Component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="details"
      >
        <div
          class="input"
        >
          <button
            type="button"
          >
            <img
              alt="back-button"
              src="arrow-8.png"
            />
          </button>
          <input
            type="text"
          />
          <button
            type="button"
          >
            <img
              alt="mic-button"
              src="microphone.png"
            />
          </button>
          <button
            type="button"
          >
            <img
              alt="settings-button"
              src="gear.png"
            />
          </button>
        </div>
        <ul
          class="detailsList"
        >
          <li>
             
            One
             
            is worth
          </li>
          <li>
            <span>
              1.09313
            </span>
             
            <span
              class="curName"
            >
              US dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1.444408
               
            </span>
             
            <span
              class="curName"
            >
              Canadian dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1
            </span>
             
            <span
              class="curName"
            >
              euros
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.000036
            </span>
             
            <span
              class="curName"
            >
              bitcoin
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.980155
            </span>
             
            <span
              class="curName"
            >
              Swiss franc
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              11.617312
            </span>
             
            <span
              class="curName"
            >
              Swedish krona
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              11.832592
            </span>
             
            <span
              class="curName"
            >
              Norwegian Krone
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              7.472205
            </span>
             
            <span
              class="curName"
            >
              Danish Krone
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1.636913
            </span>
             
            <span
              class="curName"
            >
              Australian dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1.779618
            </span>
             
            <span
              class="curName"
            >
              New Zealand dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              8.559155
            </span>
             
            <span
              class="curName"
            >
              Hong Kong dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              33.890867
            </span>
             
            <span
              class="curName"
            >
              New Taiwan dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1.477749
            </span>
             
            <span
              class="curName"
            >
              Singapore dollars
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              7.84857
            </span>
             
            <span
              class="curName"
            >
              Chinese renminbi
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              157.094163
            </span>
             
            <span
              class="curName"
            >
              Japanese yen
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1430.541284
            </span>
             
            <span
              class="curName"
            >
              South Korean won
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              89.620831
            </span>
             
            <span
              class="curName"
            >
              Indian rupee
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              18.761289
            </span>
             
            <span
              class="curName"
            >
              Mexican peso
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              5.228562
            </span>
             
            <span
              class="curName"
            >
              Brazilian real
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.905352
            </span>
             
            <span
              class="curName"
            >
              Cayman Island dollar
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.860808
            </span>
             
            <span
              class="curName"
            >
              Gibraltar pound
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.77558
            </span>
             
            <span
              class="curName"
            >
              Jordanian dinar
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.419159
            </span>
             
            <span
              class="curName"
            >
              Omani rial
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.410442
               
            </span>
             
            <span
              class="curName"
            >
              Bahraini dinar
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              0.33616
            </span>
             
            <span
              class="curName"
            >
              Kuwaiti dinar
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              4.453097
            </span>
             
            <span
              class="curName"
            >
              Polish z≈Çoty
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              38.533243
            </span>
             
            <span
              class="curName"
            >
              Thai baht
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              3.966997
            </span>
             
            <span
              class="curName"
            >
              Israeli new shekel
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              16446.471886
            </span>
             
            <span
              class="curName"
            >
              Indonesian rupiah
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              23.730877
            </span>
             
            <span
              class="curName"
            >
              Czech koruna
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              4.014963
            </span>
             
            <span
              class="curName"
            >
              UAE dirham
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              27.631384
            </span>
             
            <span
              class="curName"
            >
              Turkish lira
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              371.413274
            </span>
             
            <span
              class="curName"
            >
              Hungarian forint
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              874.034345
            </span>
             
            <span
              class="curName"
            >
              Chilean peso
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              4.099536
            </span>
             
            <span
              class="curName"
            >
              Saudi riyal
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              60.958448
            </span>
             
            <span
              class="curName"
            >
              Philippine peso
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              5.112028
            </span>
             
            <span
              class="curName"
            >
              Malaysian ringgit
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              4454.18946
            </span>
             
            <span
              class="curName"
            >
              Colombian peso
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              92.506186
            </span>
             
            <span
              class="curName"
            >
              Russian ruble
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              4.973091
            </span>
             
            <span
              class="curName"
            >
              Romanian leu
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              3.942886
            </span>
             
            <span
              class="curName"
            >
              Peruvian sol
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              1.962392
            </span>
             
            <span
              class="curName"
            >
              Bulgarian lev
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
          <li>
            <span>
              275.463714
            </span>
             
            <span
              class="curName"
            >
              Argentine peso
            </span>
            <img
              alt="right-arrow"
              class="right-details"
              src="arrow-4.png"
            />
          </li>
        </ul>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="details"
    >
      <div
        class="input"
      >
        <button
          type="button"
        >
          <img
            alt="back-button"
            src="arrow-8.png"
          />
        </button>
        <input
          type="text"
        />
        <button
          type="button"
        >
          <img
            alt="mic-button"
            src="microphone.png"
          />
        </button>
        <button
          type="button"
        >
          <img
            alt="settings-button"
            src="gear.png"
          />
        </button>
      </div>
      <ul
        class="detailsList"
      >
        <li>
           
          One
           
          is worth
        </li>
        <li>
          <span>
            1.09313
          </span>
           
          <span
            class="curName"
          >
            US dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1.444408
             
          </span>
           
          <span
            class="curName"
          >
            Canadian dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1
          </span>
           
          <span
            class="curName"
          >
            euros
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.000036
          </span>
           
          <span
            class="curName"
          >
            bitcoin
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.980155
          </span>
           
          <span
            class="curName"
          >
            Swiss franc
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            11.617312
          </span>
           
          <span
            class="curName"
          >
            Swedish krona
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            11.832592
          </span>
           
          <span
            class="curName"
          >
            Norwegian Krone
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            7.472205
          </span>
           
          <span
            class="curName"
          >
            Danish Krone
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1.636913
          </span>
           
          <span
            class="curName"
          >
            Australian dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1.779618
          </span>
           
          <span
            class="curName"
          >
            New Zealand dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            8.559155
          </span>
           
          <span
            class="curName"
          >
            Hong Kong dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            33.890867
          </span>
           
          <span
            class="curName"
          >
            New Taiwan dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1.477749
          </span>
           
          <span
            class="curName"
          >
            Singapore dollars
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            7.84857
          </span>
           
          <span
            class="curName"
          >
            Chinese renminbi
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            157.094163
          </span>
           
          <span
            class="curName"
          >
            Japanese yen
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1430.541284
          </span>
           
          <span
            class="curName"
          >
            South Korean won
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            89.620831
          </span>
           
          <span
            class="curName"
          >
            Indian rupee
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            18.761289
          </span>
           
          <span
            class="curName"
          >
            Mexican peso
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            5.228562
          </span>
           
          <span
            class="curName"
          >
            Brazilian real
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.905352
          </span>
           
          <span
            class="curName"
          >
            Cayman Island dollar
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.860808
          </span>
           
          <span
            class="curName"
          >
            Gibraltar pound
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.77558
          </span>
           
          <span
            class="curName"
          >
            Jordanian dinar
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.419159
          </span>
           
          <span
            class="curName"
          >
            Omani rial
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.410442
             
          </span>
           
          <span
            class="curName"
          >
            Bahraini dinar
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            0.33616
          </span>
           
          <span
            class="curName"
          >
            Kuwaiti dinar
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            4.453097
          </span>
           
          <span
            class="curName"
          >
            Polish z≈Çoty
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            38.533243
          </span>
           
          <span
            class="curName"
          >
            Thai baht
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            3.966997
          </span>
           
          <span
            class="curName"
          >
            Israeli new shekel
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            16446.471886
          </span>
           
          <span
            class="curName"
          >
            Indonesian rupiah
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            23.730877
          </span>
           
          <span
            class="curName"
          >
            Czech koruna
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            4.014963
          </span>
           
          <span
            class="curName"
          >
            UAE dirham
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            27.631384
          </span>
           
          <span
            class="curName"
          >
            Turkish lira
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            371.413274
          </span>
           
          <span
            class="curName"
          >
            Hungarian forint
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            874.034345
          </span>
           
          <span
            class="curName"
          >
            Chilean peso
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            4.099536
          </span>
           
          <span
            class="curName"
          >
            Saudi riyal
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            60.958448
          </span>
           
          <span
            class="curName"
          >
            Philippine peso
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            5.112028
          </span>
           
          <span
            class="curName"
          >
            Malaysian ringgit
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            4454.18946
          </span>
           
          <span
            class="curName"
          >
            Colombian peso
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            92.506186
          </span>
           
          <span
            class="curName"
          >
            Russian ruble
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            4.973091
          </span>
           
          <span
            class="curName"
          >
            Romanian leu
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            3.942886
          </span>
           
          <span
            class="curName"
          >
            Peruvian sol
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            1.962392
          </span>
           
          <span
            class="curName"
          >
            Bulgarian lev
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
        <li>
          <span>
            275.463714
          </span>
           
          <span
            class="curName"
          >
            Argentine peso
          </span>
          <img
            alt="right-arrow"
            class="right-details"
            src="arrow-4.png"
          />
        </li>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
